srothenburger1/phlex
# Faith/landing page

#### What does this PR do?

Adds a Landing page to the Phlex application.

Description of Task to be completed?
- Create a landing page with a submit button and enter keypress

#### How should this be manually tested?
- Pull the branch
- run `npm install` to install dependencies
- run `npm start` to start the application
- You should see the landing page in the browser window.

####  Any background context you want to provide?

N/A

 #### What are the relevant Phlex Board stories?

[#card-18389765](https://github.com/srothenburger1/phlex/projects/1#card-18389765)

####  Any Screenshots?
![Screen Shot 2019-03-11 at 12 51 01 AM](https://user-images.githubusercontent.com/26274319/54105145-d12caf00-4397-11e9-9cbe-b02559135bf8.png)


 #### Any Questions?
N/A
